instrumentation-genai/opentelemetry-instrumentation-openai-v2/tests/test_async_chat_completions.py

```diff
@@ -633,6 +633,55 @@ async def test_async_chat_completion_multiple_tools_streaming_no_content(
     )
 
 
+@pytest.mark.vcr()
+@pytest.mark.asyncio()
+async def test_async_chat_completion_streaming_sample_out(
```

Comment:
note for later (so feel free to resolve), we can re-use cassettes when the server's response is uninteresting vs another code path. This often exists between "async" and not.

I've made a fixture like this to handle it, which makes the diffs a lot shorter

```python
@pytest.fixture
def vcr_cassette_name(request):
    """
    Strips `_async` from the test function name as they use the same data.
    """
    # Get the name of the test function
    test_name = request.node.name

    # Remove '_async' from the test name
    cassette_name = test_name.replace("_async", "")

    return cassette_name
```

Reply:
I like the idea, but  it won't apply to this PR - I test no-streaming+sync and streming+async, so recordings are different.